azure-devops/golang/stages/40-delivery/arm.yaml

```diff
@@ -46,11 +46,28 @@ stages:
               overrideParameters: '-location $(location) -environment $(environment) -functionAppName $(functionAppName) -inboundAllowedIps $(inboundAllowedIps)'
               templateLocation: 'Linked artifact'
               deploymentMode: 'Incremental'
+              deploymentOutputs: 'OUTPUTS'
           - task: 'AzureFunctionApp@1'
             inputs:
               appName: '$(environment)-fn-$(functionAppName)-$(location)'
               appType: 'functionAppLinux'
               azureSubscription: '$(serviceConnection)'
               package: '$(Build.ArtifactStagingDirectory)/$(Build.BuildId).zip'
               deployMethod: 'zipDeploy'
+          - script: |
+              if grep -r 'goose' db/*.sql; then
```

Comment:
Very good. Let me suggest something to be even better? Place this snippet here in another file, and include the fille here. What do you think? You can also think if it's a good idea to use the Azure features to do the IF instead of doing it in the bash command.

Reply:
Ok, my friend.